e2e/cluster/local/docker-compose.yaml

```diff
@@ -0,0 +1,41 @@
+version: "3.7"
+
+volumes:
+  kubeconfig:
+    name: e2e-local-kubeconfig
+
+services:
+
+  leader:
+    container_name: local-leader
+    image: "e2e/cluster/local/k3s"
+    build:
+      context: ./images/k3s/.
+      args:
+        - ARCH=${ARCH}
+    command: server --no-deploy=traefik,metrics-server
+    environment:
+      - K3S_TOKEN=e2e
+    hostname: local-leader
+    privileged: true
+    ports:
+      - "172.17.0.1:6443:6443" # k3s
```

Comment:
Will the address be always 172.17.0.1 for the host (dapper container i assume?)

Reply:
It's the default for docker0. This worked for me on a linux box, my mac, and it passed drone so that's good enough for me for now. I do agree that this should be more flexible in the future but I just wanted to get it off my plate.